source-code/cli/src/commands/machine/translate.ts

```diff
@@ -16,11 +18,11 @@ async function translateCommandAction() {
 	// Set up the environment functions
 	const env: InlangEnvironment = {
 		$import: initialize$import({
-			// @ts-ignore TODO: use @inlang-git/fs
+			// TODO: use @inlang-git/fs
```

Comment:
You don't need to use `@inlang-git/fs`. `@inlang-git/fs` is for browser environments. The CLI operates in node